src/main/java/org/gridsuite/shortcircuit/server/service/ShortCircuitService.java

```diff
@@ -118,7 +121,9 @@ public ShortCircuitAnalysisResult getResult(UUID resultUuid, FaultResultsMode mo
                 break;
         }
         if (result.isPresent()) {
-            ShortCircuitAnalysisResult res = result.map(r -> fromEntity(r, mode)).orElse(null);
+            Optional<ShortCircuitAnalysisResultEntity> sortedResult = Optional.of(sortByElementId(result.get()));
```

Comment:
I don't understand the necessity to use optional here

Reply:
because in the next instruction we used an Optionnal to do a map on it

Reply:
you can change this next instruction from 
```
Optional<ShortCircuitAnalysisResultEntity> sortedResult = Optional.of(sortByElementId(result.get()));
ShortCircuitAnalysisResult res = sortedResult.map(r -> fromEntity(r, mode)).orElse(null);
```

to somethings like 

```
ShortCircuitAnalysisResultEntity sortedResult = sortByElementId(result.get());
ShortCircuitAnalysisResult res = fromEntity(sortedResult, mode)
```

I think the code would be cleaner